docs/_docs/01_getting-started/install.md

```diff
@@ -23,6 +23,15 @@ You can install Terragrunt on macOS using [Homebrew](https://brew.sh/): `brew in
 
 ### Linux
 
+There are multiple ways to install Terragrunt on Linux depending on your distribution.
+
+#### Arch Linux
+
+  * [`aur/terragrunt`](https://aur.archlinux.org/packages/terragrunt)
+  * [`aur/terragrunt-bin`](https://aur.archlinux.org/packages/terragrunt-bin)
```

Comment:
```suggestion
  * [`aur/terragrunt`](https://aur.archlinux.org/packages/terragrunt): Build terragrunt from source using a versioned release.
  * [`aur/terragrunt-bin`](https://aur.archlinux.org/packages/terragrunt-bin): Fetch a prebuilt binary from a versioned release.
```

Reply:
This isn't super typical for Arch Linux users (the `*-bin` suffix is a well known convention), but I'm happy to add the additional context!

Reply:
Ah I didn't realize! Thanks for the clarification and now I know. To streamline, will keep it in and it can't hurt.